river/cluster/textclust.py

```diff
@@ -0,0 +1,643 @@
+import math
+
+import numpy as np
+import pandas as pd
+
+from river import base
+
+__all__ = ["TextClust"]
+
+
+class TextClust(base.Clusterer):
+    r"""
+
+    textClust [^1][^2]
+
+    textClust is a stream clustering algorithm for textual data that can identify and track topics
```

Comment:
```suggestion
    r"""textClust, a clustering algorithm for text data. 

    textClust[^1][^2] is a stream clustering algorithm for textual data that can identify and track topics
```

Reply:
So do I need to fix your comments first or how do we proceed?

Reply:
Yes you can fix these first I would say :)